addons/auth/package.json

```diff
@@ -21,52 +21,52 @@
     "doc:toc": "doctoc README.md"
   },
   "dependencies": {
-    "@embroider/test-setup": "^0.48.1",
-    "ember-auto-import": "^2.4.2",
```

Comment:
Does `ember-auto-import` not need to be a `dependency`? The [docs](https://github.com/ef4/ember-auto-import#installing-ember-auto-import-in-an-addon) say addons should have it as a `dependency`

Or do we not care because it's consumed by only our application which we know will have it?

Reply:
By default, the addon blueprint makes it a devDep.

Reply:
My 2 cents:
- Indeed, by default the blueprint adds it as devDep.
- When using `ember-auto-import` in an addon that will be consumed by Ember app, the documentation strongly recommends to add it as dependency to be present for the app that is consuming it. If we do not do such thing, we can run [in this linting issue](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-require.md), we will not necessarily run on it, but most likely. I run into this while experimenting with Ember 4 stuff.